tests/dut_console/test_idle_timeout.py

```diff
@@ -14,7 +14,8 @@
 ]
 
 
-def test_timeout(duthost_console, duthost):
+def test_timeout(duthost_console, duthosts, enum_supervisor_dut_hostname):
+    duthost = duthosts[enum_supervisor_dut_hostname]
```

Comment:
Why we need duthost here?

Reply:
line 29 uses this duthost.
for chassis only sup has console link